src/authorization/graphql/user.graphql

```diff
@@ -49,8 +48,35 @@ permissions: [String]!,
 operation: OperationType
 }
 
+input UserInputFilter {
+  search: UserSearchInput
+}
+
+input UserSearchInput {
+  and: UserSearchCondition
+  or: UserSearchCondition
+}
+
+input UserSearchCondition {
+  email: SearchCondition
+  firstName: SearchCondition
+  middleName: SearchCondition
+  lastName: SearchCondition
+}
+
+input SearchCondition {
```

Comment:
Can we make this `StringSearchCondition`? This would vary for other types like `IntSearchCondition` or `FloatSearchCondition`